extensions/pkg/controller/utils.go

```diff
@@ -128,20 +127,6 @@ func DeleteAllFinalizers(ctx context.Context, client client.Client, obj client.O
 	})
 }
 
-// SecretReferenceToKey returns the key of the given SecretReference.
-func SecretReferenceToKey(ref *corev1.SecretReference) client.ObjectKey {
```

Comment:
Some extensions might depend on the removed functions.
Can you add an alias similar to this one?
```go
var TryPatch = kutil.TryPatch
```

Reply:
Good idea, added one for `GetSecretByReference` since this one is definitely used, `SecretReferenceToKey` is no longer needed in g/g and not used by any extensions (did a search in the entire gardener org).